server/src/lib/model/db/migrations/20200422152745-cascade-skipped-sentence-deletion.ts

```diff
@@ -0,0 +1,12 @@
+export const up = async function (db: any): Promise<any> {
+  return db.runSql(
+    `
+    ALTER TABLE skipped_sentences DROP FOREIGN KEY skipped_sentences_ibfk_1;
+    ALTER TABLE skipped_sentences ADD CONSTRAINT skipped_sentences_ibfk_1 FOREIGN KEY (sentence_id) REFERENCES sentences (id) ON DELETE CASCADE;
```

Comment:
I'm not sure if this is strictly necessary, but in `import-sentences` we do a bulk deletion of unused sentences that don't match a current version and I thought it was possible that that could deadlock at some point if we try to delete a sentence that's referenced in `skipped_sentences` (but not in anything else, where it wouldn't affect `is_used`).